spec/core/ics-004-channel-and-packet-semantics/README.md

```diff
@@ -1099,6 +1113,11 @@ function timeoutOnClose(
     // delete our commitment
     provableStore.delete(packetCommitmentPath(packet.sourcePort, packet.sourceChannel, packet.sequence))
 
+    // if there are no in-flight packets on our end, we can automatically go to FLUSHCOMPLETE
+    if channel.flushStatus === FLUSHING && pendingInflightPackets(portIdentifier, channelIdentifier) == nil {
+        channel.flushState = FLUSHCOMPLETE
+    }
+
```

Comment:
Is this necessary? The counterparty channel is closed. Might be good to spend a little time thinking about the cases when a channel closes during an upgrade